src/server/controller/CurvytronController.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Curvytron Controller
+ *
+ * @param {Collection} clients
+ */
+function CurvytronController(clients) {
+    // create socket for all clients
+    this.clientGroup = new SocketGroup(clients);
```

Comment:
Sorry, I thought the `RoomsController` was (by name) linked to the `rooms` page, and I wanted to handle the total players in a global controller. In my defense, in the client folder there is a `RoomsController` and a `CurvytronController` :)